static/js/zamboni/devhub.js

```diff
@@ -990,7 +990,10 @@ function initCatFields(delegate) {
             $("input", $misc).prop("checked", checkedLength <= 0);
         };
         var checkOther = function() {
-            $("input", $main).prop("checked", false).prop("disabled", false);
+          // Save Android categories
+          var $androidCat = $main.find('input[name="form-1-categories"]');
+          $("input", $main).prop("checked", false).prop("disabled", false);
+          $($androidCat+":checked").prop("checked", true).prop("disabled", false);
```

Comment:
In this version of jquery you can do `$($androidCat.selector + ":checked")` but otherwise concatenating a JQuery object with a string is going to cause an error.